SPECS-EXTENDED/netcdf/netcdf.spec

```
@@ -398,6 +398,11 @@ done
 
 
 %changelog
+* Tue May 21 2024 George Mileka <gmileka@microsoft.com> - 4.9.0-6
+  Remove the explicit version in build/runtime dependency on hdf5 to unblock
+  the arm64 build.
```

The explicit version is OK, the usage of the undefined `_hdf5_version` is not. Let's reword this a bit to clarify the issue:
```suggestion
  Remove the use of the '_hdf5_version' macro from the build-time dependencies.
```

---

How about:
`Remove the recently added macro version and explicit version for hdf5 dependencies to unblock the arm64'